### DISCUSSION TITLE
Error while using supabase db remote commit

### DISCUSSION THREAD

**Discussion by rbkayz** [supabase/supabase#8472] [General]
Hi community,

Facing an issue when I try to use "supabase db remote commit"

It returns an: Error: unexpected end of JSON input

Breaking my head over this. Could anyone help please?

Thanks

**Comment by vishwasnavadak** [comment]
I ran into this issue as well just now but with the supabase diff command. 

```
supabase db diff init_db -f init_db
```
While i was following the instructions from https://supabase.com/docs/guides/cli/local-development

**Comment by vishwasnavadak** [comment]
As per this issue (https://github.com/supabase/cli/issues/369), it is fixed on the newer version of the supabase cli (v1.0.3 or above).
